backend/backend.js

```diff
@@ -16,26 +16,43 @@ app.get("/", (req, res) => {
 });
 
 app.get("/movies/popular", async (req, res) => {
+  listOfPopMovies = [];
   try {
     const result = await axios.request(
       "https://api.themoviedb.org/3/movie/popular?api_key=" +
         process.env.API_KEY +
         "&language=en-US&page=1&region=US"
     );
-    res.send(result.data.results);
+
+    listOfPopMovies.push(...result.data.results);
```

Comment:
Out of curiosity what do these lines do as opposed to the original code?

Reply:
The code that you had a comment on just makes a second request for the second page and adds it onto the current list of movies